app/proc/telegram.go

```diff
@@ -160,7 +161,7 @@ type recipient struct {
 }
 
 func (r recipient) Recipient() string {
-	if !strings.HasPrefix(r.chatID, "@") {
+	if _, err := strconv.ParseInt(r.chatID, 10, 64); err != nil && !strings.HasPrefix(r.chatID, "@") {
```

Comment:
I'm not sure what exactly it's preserving (non-numeric chat_id without @ I guess) but some tests for numeric ids passing (or not) will be nice to have